src/applications/gi/components/vet-tec/VetTecApprovedPrograms.jsx

```diff
@@ -49,7 +49,6 @@ class VetTecApprovedPrograms extends React.Component {
           <td>
             <div className="form-radio-buttons gids-radio-buttons">
               <input
-                id={`radio-${index}`}
```

Comment:
Looking at this markup, I think the issue is the `input[type="radio"]` is being labelled by the column header and the label using `aria-labelledby` on line 61 below.

Removing the native semantics of the index ID + label htmlFor goes against the [first rule of ARIA](https://www.w3.org/TR/using-aria/#rule1). I'd prefer to remove the `aria-labelledby`, restore the `id` and retest. Since this is a table, screen readers should announce "Program" correctly when the cell is settled into.

Let's update the table and re-test in Staging, assuming no regressions.